.github/workflows/print-topology.yml

```
@@ -12,6 +12,11 @@ on:
         type: string
         required: false
         default: 'docs/topology/stream.txt'
+      allTopicsFilePath:
```

```suggestion
      processorTopicsOutputFilePath:
```

Should be easier to understand. What we gather here is "(all) the topics used in processor". All is too broad. Also I would make it easier to understand that it's path where to store extracted topics, rather than where to look for them.